tests/test_utils.py

```diff
@@ -18,10 +18,136 @@
+    preserve_index (:obj:`bool`, `optional`, defaults to :obj:`True`): Whether to store the index as an additional column in the resulting ``Table``. The default of None will store the index as a column, except for RangeIndex which is stored as metadata only. Use ``preserve_index=True`` to force it to be stored as a column.
+    nthreads (:obj:`int`, defaults to :obj:`None`): If greater than 1, convert columns to Arrow in parallel using indicated number of threads.
+    columns (:obj:`list`, `optional`): List of column to be converted. If None, use all columns.
+    safe (:obj:`bool`, defaults to :obj:`True`): Check for overflows or other unsafe conversions.
+
+Returns:
+    :obj:`Table`: New table without the columns.
+
+Raises:
+    KeyError: If any of the passed columns name are not existing.
+    ValueError
+
+Example::
+    >>> import pandas as pd
+    >>> import pyarrow as pa
+    >>> df = pd.DataFrame({
+        ...     'int': [1, 2],
+        ...     'str': ['a', 'b']
+        ... })
+    >>> pa.Table.from_pandas(df)
+    <pyarrow.lib.Table object at 0x7f05d1fb1b40>"""
+
+        self.assertEqual(convert_numpydoc_to_groupsdoc(original_numpydocstring), expected_conversion)
+
+    def test_convert_parametype_numpydoc_to_groupsdoc(self):
+        # test canonical
+        original_numpydocstring = "bool, optional, default True"
+        expected_conversion = ":obj:`bool`, `optional`, defaults to :obj:`True`"
+        self.assertEqual(convert_parametype_numpydoc_to_groupsdoc(original_numpydocstring), expected_conversion)
+
+        # test without `optional`
+        original_numpydocstring = "bool, default True"
+        expected_conversion = ":obj:`bool`, defaults to :obj:`True`"
+        self.assertEqual(convert_parametype_numpydoc_to_groupsdoc(original_numpydocstring), expected_conversion)
+
+        # test with extra info
+        original_numpydocstring = "bool, optional, default True (some extra info)"
+        expected_conversion = ":obj:`bool`, `optional`, defaults to :obj:`True`"
+        self.assertEqual(convert_parametype_numpydoc_to_groupsdoc(original_numpydocstring), expected_conversion)
+
+        # test int
+        original_numpydocstring = "int, optional, default 1"
+        expected_conversion = ":obj:`int`, `optional`, defaults to 1"
+        self.assertEqual(convert_parametype_numpydoc_to_groupsdoc(original_numpydocstring), expected_conversion)
+
+        # test str
+        original_numpydocstring = "str, optional, default 'some_str'"
+        expected_conversion = ":obj:`str`, `optional`, defaults to 'some_str'"
```

Comment:
Here we should have the backticks for 'some_str'

Reply:
Should it be 
```
":obj:`str`, `optional`, defaults to `'some_str'`"
```

or

```
":obj:`str`, `optional`, defaults to :obj:`'some_str'`"
```
because of [this](https://github.com/huggingface/doc-builder/pull/89#discussion_r804670648)
?

Reply:
Oh you're right, we need the obj too!

Reply:
handled in https://github.com/huggingface/doc-builder/pull/89/commits/d4a375281ca6cfb3182bc2bcc5006784c18bba42